notebooks/tutorials/1_getting_started_lesson4/notebook.ipynb

```diff
@@ -0,0 +1,406 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Strategy Definition"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have learned how to access and manipulate data in Quantopian, let's construct a data pipeline for our long-short equity strategy. In general, long-short strategies consist of modeling the relative value of assets with respect to each other, and placing bets on the sets of assets that we are confident will increase ([long](https://www.investopedia.com/terms/l/long.asp)) and decrease ([short](https://www.investopedia.com/terms/s/short.asp)) the most in value.  \n",
+    "\n",
+    "Long-short equity strategies profit as the spread in returns between the sets of high and low value assets increases. Their performance relies entirely on the quality of their underlying ranking model. In this tutorial we will use a simple ranking schema for our strategy:  \n",
```

Comment:
It's not that the assets themselves are "high and low value", it's that their factor values are high and low.

Reply:
`Their performance relies entirely on the quality of their underlying ranking model.` -> `The quality of long-short equity strategy relies entirely on the quality of its underling ranking model.`